source/ota_http.c

```diff
@@ -51,7 +51,7 @@ OtaErr_t initFileTransfer_Http( OtaAgentContext_t * pAgentCtx )
     OtaFileContext_t * fileContext = NULL;
 
     LogDebug( ( "Invoking initFileTransfer_Http" ) );
-    assert( pAgentCtx != NULL && pAgentCtx->pOtaInterface != NULL );
+    assert( pAgentCtx != NULL && pAgentCtx->pOtaInterface != NULL && pAgentCtx->pOtaInterface->http.init != NULL );
```

Comment:
Minor ( ( ) && ( ) .. will make it slightly more readable.

Reply:
I have added those changes to the last commit.